cmd/mattermost/commands/webhook.go

```diff
@@ -262,6 +270,31 @@ func createOutgoingWebhookCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func deleteWebhookCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("WebhookID is not specified")
+	}
+
+	webhookId := args[0]
+	errIncomingWebhook := app.DeleteIncomingWebhook(webhookId)
+	errOutgoingWebhook := app.DeleteOutgoingWebhook(webhookId)
+
+	if errIncomingWebhook != nil && errOutgoingWebhook != nil {
+		if errIncomingWebhook != nil {
+			return errors.New("Unable to delete webhook '" + webhookId + "'")
+		}
+		return errors.New("Unable to delete webhook '" + webhookId + "'" + " ")
```

Comment:
I don't understand why this return a different error (simply concatenating a space at the end). My proposal here would be leave it like this:
```go
if errIncomingWebhook != nil && errOutgoingWebhook != nil {
	return errors.New("Unable to delete webhook '" + webhookId + "'")
}
```

Reply:
I was earlier printing the errIncomingWebhook.Error() and errOutgoingWebhook.Error() for debugging some error. But now it doesn't make any sense to keep two.